zaproxy/src/org/zaproxy/zap/extension/attacksurfacedetector/AttackSurfaceDetector.java

```diff
@@ -112,11 +102,6 @@ public URL getURL()
             return null;
         }
     }
-    @Override
-    public boolean isEnabled()
-    {
-        return true;
```

Comment:
This is unnecessary, also problematic if the user actually disabled the addon via Options.